server/world/biome/biome.go

```diff
@@ -0,0 +1,33 @@
+package biome
+
+// Biome is a region in a world with distinct geographical features, flora, temperatures, humidity ratings,
+// and sky, water, grass and foliage colors.
+type Biome interface {
+	// Temperature returns the temperature of the biome.
+	Temperature() float64
+	// Rainfall returns the rainfall of the biome.
+	Rainfall() float64
+	// String returns the biome name as a string.
+	String() string
```

Comment:
Do we strictly need this?

Reply:
No, but it can be useful in some cases. Would you like me to remove it?